vendor/engines/projects/app/views/projects/projects/index.html.haml

```diff
@@ -0,0 +1,17 @@
+= content_for :h1 do
+  = t(".head")
+
+- if current_ability.can?(:create, Projects::Project)
+  %p= link_to t(".add"), new_facility_project_path, class: "btn-add"
```

Comment:
I've started setting up some generic helper localse for these over in my other branch. We don't need to do anything right now, but we could revisit these once we get text_helpers in place.